main/acle.md

```diff
@@ -9043,9 +9115,18 @@ following it. --><span id="__arm_za_disable"></span>
 
 The intrinsics in this section have the following properties in common:
 
-*   Every argument named `tile`, `slice_offset` or `tile_mask` must
-    be an integer constant expression in the range of the underlying
-    instruction.
+*   Every argument named `tile` or `tile_mask` must be an integer constant
```

Comment:
Right.  This PR has two commits, the first from #225 and the second for this PR.  Doing both PRs against main didn't seem to make much sense when they clashed so heavily.  One PR or the other would need to be rewritten and re-reviewed once the other patch went in.

So it's probably better to review the second commit individually, rather than the “Files changed” tab.

Sorry for the confusion.  github doesn't seem to make patch series very easy to handle :(

Reply:
No worries, I'm still figuring out a bit how Github works. I'll review #225 as well so that you don't have to rebase this patch.